src/components/companyOverview.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { companySelector} from "../selectors/companySelector";
+import { tickerSelector } from "../selectors/tickerSelector";
+import { DisplayWrapper } from "./generics/displayWrapper";
+import { TitleHeader } from "./generics/titleHeader";
+import { Text } from "./generics/text";
+import { useDispatch, useSelector } from "react-redux";
+
+
+export const CompanyOverview = () => {
+  //Call necessary selectors for display data
+  const dispatch = useDispatch();
+    const companyInfo = useSelector(companySelector);
+    const ticker = useSelector(tickerSelector);
+  return (
+    <DisplayWrapper width="25%" mr="auto" ml="auto">
```

Comment:
Should mr and ml as auto be the default on the DisplayWrapper? Seems like we use that a lot.